docker-storage-setup.sh

```diff
@@ -435,11 +438,11 @@ setup_lvm_thin_pool () {
   tpool=`get_configured_thin_pool`
 
   if [ -n "$tpool" ]; then
-     local escaped_pool_lv_name=`echo $POOL_LV_NAME | sed 's/-/--/g'`
+     local escaped_pool_lv_name=`echo $CONTAINER_THINPOOL | sed 's/-/--/g'`
```

Comment:
I have changed the code again to just check if the STORAGE_IN_FILE is /etc/sysconfig/docker-storage-setup then it will default CONTAINER_THINPOOL to docker-pool if not set.